src/patternfly/components/Nav/nav.scss

```diff
@@ -260,76 +222,12 @@
 
   // Divider
   .pf-c-divider {
-    // Update during breaking change
-    padding-left: var(--pf-c-nav__c-divider--MarginLeft);
-
-    // end - Update during breaking change
+    padding-right: var(--pf-c-nav--c-divider--PaddingRight);
```

Comment:
why is padding being set to 0 here ? is it to override ? should it just be the value 0

Reply:
It's set once here so that it can inherit padding values in different contexts, or be themed.